### DISCUSSION TITLE
cast to device

### DISCUSSION THREAD

**Discussion by xdken** [smplayer-dev/smplayer#1073] [General]
i just found out when i right click on a video there is a option called "cast to device" that allows me to send video to tv (same network).
Can Smplayer do that ? My goal is to use subtitle and frame interpolation [svp4](https://www.svp-team.com/). 
It's just so good lmao just add it pls

**Comment by bendover22** [comment]
If question is, does smp have "cast to device" - it does.
I don't use it, but in smp, R-click open vid > Play, at bottom of menu is "cast to".  Probably, avail options depend on (recognized) equip. attached to your system.
